### PR TITLE
Add tags to the Understanding ECMAScript posts

### PR DESCRIPTION
**Pull request by marjakh**
Also unify the posts a bit (add "Summary" to the ones where it was missing etc).

### CODE REVIEW

File: src/blog/understanding-ecmascript-part-1.md [hunk 1]
```diff
@@ -6,9 +6,13 @@ avatars:
 date: 2020-02-03 13:33:37
 tags:
   - ECMAScript
+  - Understanding ECMAScript
```
Comment (RReverser) [inline src/blog/understanding-ecmascript-part-1.md:9]: Huh, do tags with spaces actually work and render fine?

Comment (marjakh) [reply src/blog/understanding-ecmascript-part-1.md:9]: They seem to "work" as in they will get transformed to the space-free version. I can also use that explicitly if you'd prefer.

Comment (marjakh) [reply src/blog/understanding-ecmascript-part-1.md:9]: Ok this comment was inaccurate. So, they render as UnderstandingECMAScript and the link is /blog/tags/understanding-ecmascript

So maybe it's fine? I'd like the link to be understanding-ecmascript and not understandingecmascript.

Comment (RReverser) [reply src/blog/understanding-ecmascript-part-1.md:9]: Yeah the link sounds good. The rendering without spaces is a bit weird, but if you're comfortable with it, fine by me (I suppose we can fix it later if it's a problem).

**Pull request merged**